data/transactions/logic/opcodes.go

```diff
@@ -21,7 +21,10 @@ import (
 )
 
 // LogicVersion defines default assembler and max eval versions
-const LogicVersion = 6
+const LogicVersion = 7
```

Comment:
Should I bump up the version like I'm doing here? I'm assuming that this is going out _after_ v. 6 and that new code should therefore have a higher version.